ignite/metrics/metric.py

```diff
@@ -300,7 +300,7 @@ def compute(self):
     _required_output_keys = required_output_keys
 
     def __init__(
-        self, output_transform: Callable = lambda x: x, device: Union[str, torch.device] = torch.device("cpu")
+        self, output_transform: Callable = lambda x: x, device: Union[str, torch.device] = torch.device("cpu"), skip_unrolling : bool =False
```

Comment:
Let's also update above docstring by adding the new argument. Please also check CONTRIBUTING guideline the part about how to add `.. versionadded::` tag in the bottom of the docstring. Version to put should be `0.5.1`

Reply:
Should I add ``` .. versionchanged::``` instead?

Reply:
Yes, correct, it should be versionchanged tag, https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md#writing-documentation

Reply:
On it. Thanks.

Reply:
Have made the changes. Kindly review.